public/modules/custom/grants_oma_asiointi/src/Controller/GrantsOmaAsiointiController.php

```diff
@@ -103,6 +103,16 @@ public static function create(ContainerInterface $container): GrantsMandateContr
     );
   }
 
+  /**
+   * Controller for setting time for closing a notification.
+   */
+  public function logCloseTime(string $closeTime) {
+    $this->grantsProfileService->setNotificationShown($closeTime);
+
+    // Return a JSON response with the logged close time.
+    return new JsonResponse(['closeTime' => $closeTime]);
```

Comment:
JsonResponse:lta puuttuu use statement tiedoston alkupäästä:

![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/43133397/206be106-5599-4bb2-b2e0-4fcfffa3a42f)